src/LeanCode.ContractsGenerator/ProjectLoader.cs

```diff
@@ -1,75 +1,75 @@
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
-using Buildalyzer;
-using Buildalyzer.Workspaces;
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.MSBuild;
 
 namespace LeanCode.ContractsGenerator
 {
-    public class ProjectLoader
+    public class ProjectLoader : IDisposable
     {
-        private static readonly IReadOnlySet<string> AllowedPackaged = new HashSet<string>
-        {
-            "LeanCode.CQRS",
-            "LeanCode.Time",
-        };
-
         private readonly CSharpCompilationOptions options;
 
-        private readonly AnalyzerManager manager;
-        private readonly List<IProjectAnalyzer> projects;
+        private readonly MSBuildWorkspace msbuildWorkspace = MSBuildHelper.CreateWorkspace();
+        private readonly List<Project> projects = new();
 
         public ProjectLoader(CSharpCompilationOptions options)
         {
             this.options = options;
 
-            manager = new AnalyzerManager(null);
-            projects = new List<IProjectAnalyzer>();
+            msbuildWorkspace.WorkspaceFailed += static (sender, args) => Console.Error.WriteLine(args.Diagnostic.Message);
```

Comment:
I'm... against this? Would rather prefer gathering the errors and then throwing, leaving error reporting to the parent method. That would improve separation of concerns.

Reply:
I'll just remove it for now.